optaplanner-docs/pom.xml

```diff
@@ -11,7 +11,7 @@
   </parent>
 
   <artifactId>optaplanner-docs</artifactId>
-  <packaging>jar</packaging>
+  <packaging>pom</packaging>
```

Comment:
Are you sure this doesn't cause trouble for optaplanner-distribution?
Presuming that target/*.zip file still contains the docs, this PR is fine.